tests/unit/test_repo_manager.py

```diff
@@ -179,3 +179,30 @@ def testRemoveVariantSet(self):
         with self.assertRaises(exceptions.RepoManagerException):
             self.repoManager.removeVariantSet(
                 'dataset1', paths.variantSetName)
+
+    def testAddBioSample(self):
```

Comment:
also add entries in the end to end repo manager test: `tests/end_to_end/test_repo_manager.py`